test/validation/schema_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Litmus.Validation.SchemaTest do
+  use ExUnit.Case
+
+  alias Litmus.Type
+  alias Litmus.Validation.Schema
+
+  test "check data against defined schema" do
+    login_schema = %{
+      id: %Type.Any{
+        required: true
+      },
+      username: %Type.String{
+        min_length: 3,
+        max_length: 15,
+        regex: "^[a-zA-Z0-9_]+$",
+        required: false
```

Comment:
is `false` the default value for required if not specified?

Reply:
yes. But I just want to test both ways in future. I could delete it for now.

Reply:
no worries, I was curious about the behavior, you can leave it as is 👍